src/game/carAction.js

```diff
@@ -0,0 +1,33 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+
+const isRandomMove = () => {
+  const randNum = () => MissionUtils.Random.pickNumberInRange(0, 9);
+  if (randNum() >= 4) return true;
+  return false;
+};
+
+export const moveCarEvent = (carAndLocationList) => {
+  const movedCarList = carAndLocationList.map((car) => {
+    if (isRandomMove()) {
+      return { name: car.name, location: car.location + 1 };
+    }
+    return car;
+  });
+  return movedCarList;
+};
```

Comment:
흠🤔 뭔가 이 부분 조금 더 다듬을 수 있을 것 같은데 좋은 생각이 잘 안 나네요...
```suggestion
export const moveCarEvent = (carAndLocationList) => {
  return carAndLocationList.map(({ name, location }) => {
    return { name, location: location + isRandomMove() };
  });
};
```
이건 boolean이 숫자처럼 평가되는 것을 이용한 정말 안 좋은 코드인데요ㅋㅋㅋㅋ 이렇게 해도 동작은 할 것 같네요😅

Reply:
boolean 이 숫자처럼 작용해서 간결하게 작성할 수 있으나 가독성 측면에서 떨어질 수 있겠다는 생각이들었습니다 
덕분에 한번더 생각해볼 계기가 되었습니다 감사합니다 !!